src/app/pages/resources/overview/overview.jsx

```diff
@@ -15,6 +16,9 @@ import {
   ChplLink,
   InternalScrollButton,
 } from '../../../components/util';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
+import ArrowUpwardIcon from '@material-ui/icons/ArrowUpward';
+import NavigationIcon from '@material-ui/icons/Navigation';
```

Comment:
We don't have a linting plugin installed that does this yet, but generally the order of imports should be that external (not written by us) things come first, followed by our things. These lines should start after the imports from material-ui/core, and there should be a blank line between these and the first imports from our components

Reply:
Sounds good! I will apply this to feature files.